aws/cloudformation/privatelink-with-network-firewall.yaml

```
@@ -744,6 +744,8 @@ Outputs:
       ROSA private VPC CIDR block. Use that CIDR block for the --machine-cidr
       option of the "rosa create cluster" command.
     Value: !GetAtt rRosaVpc.CidrBlock
+    Export:
+      Name: RosaVpcCidr
```

In real life, we should ensure that exports are unique in the account/region. It is a good practice to prefix them with the stack name (if the customer deploys several clusters for example).
Anyway, you can keep it this way. We'll see if it raises issues.

```suggestion
      Name: !Join [":", [!Ref "AWS::StackName", RosaVpcCidr]]
```

---

I understand that, but that means we'll have to hardcode the stack name is the cloudformation template as ROSA-infra, so I avoided that.